torch/_subclasses/fake_tensor.py

```diff
@@ -791,6 +791,7 @@ def __init__(
         allow_non_fake_inputs=False,
         shape_env=None,
         static_shapes=None,
+        avoid_device_init=False,
```

Comment:
Do we need this as a flag ? Could we just set it unconditionally if we're trying to run cuda fake tensors with no cuda device available ?